docs/tutorials/zero-to-hero.md

```diff
@@ -308,601 +116,78 @@ export function getResponseByKey(key: string): string {
-
-Let's now wire these functions to the front end.
-
-It would be great if we could create two buttons:
-
-* the first button should get the price of BTC and save it to the blockchain
-* the second button should display the saved price of BTC to the front end
-
-### Building the front end
-
-> In the file `src/index.html`
-> - Replace the contents of `div.signed-in-flow` with the code below
-
-```html
-<h3>Hi, <i id="account-id"></i>! </h3>
-<button id="sign-out-button" class="btn btn-danger btn-lg">Sign-out</button>
-<hr>
-<button class="btn btn-primary btn-lg" onClick={makeApiCallAndSave()}>Make API call and save response to blockchain</button>
-<button class="btn btn-secondary btn-lg" onClick={fetchAndDisplayResponse()}>Retrieve response and display</button>
-<!--Creating a div to insert our API response to-->
-<hr>
-<div class="p-2 bg-success text-white" id="response"></div>
-```
-
-Now that we've created our front end elements, we need to code the functions we've set to be called `onClick`.
-
-The first button will work right away and will print the results of our API call to the console as expected.  We haven't implemented the code for the second button yet, but we will.
-
-### Wiring up our front end elements
-
-The function `makeApiCallAndSave` has already been defined, but we need to write the function `fetchAndDisplayResponse`.
-
-> In the file `src/main.js`
-> - Copy and paste the following code to the bottom of the file
-
-```js
-async function fetchAndDisplayResponse() {
-  // getting the response from the blockchain
-  let apiResponse = await contract.getResponse();
-  // logging on the console for some feedback
-  console.log(apiResponse);
-  // Displaying the message once we have it.
-  document.getElementById('response').innerText = apiResponse;
-}
-```
-
-Now clicking the "Make API Call" button will make the API call and save the response to the blockchain. (You should keep the dev console open to make sure it's happening as expected).
-
-Clicking the "Get Response And Display" button will get the response from the blockchain, and display it to the front end.
-
-Great work! You've now built an end-to-end application on the Near Protocol. Piece of cake, right?
-
-Again, feel free to click **Run** and test out the controls you've just added.
-
-### Your turn
-
-Try wiring the contract methods you created, `setResponseByKey` and `getResponseByKey`, to the front end.
-
-The front end should allow for the following functionality:
-
-* The user should be able to save a key and its corresponding response to the blockchain
-* The user should be able to input a key and retrieve the corresponding response from the blockchain
-
-How did it go?
-
-Here's our solution.
-
-**Front end**
-
-> In the file `src/index.html`
-> - We appended the following HTML to `div.signed-in-flow` (just before the closing `</div>` tag)
+Check that the application still builds and starts by running `yarn dev` from the Gitpod command line.
```

Comment:
Can we use `yarn build` instead? `yarn dev` opens up a new browser window and I'm tempted to try things out, but they're not working yet. The user can expect `yarn build` should throw no errors.

Reply:
done